From: Paul Boddie <paul@boddie.org.uk>
Date: Tue, 15 Aug 2023 23:50:33 +0200
Subject: Fix Python packaging metadata.

Adjust XStatic package versions; remove drawing-related packages.

moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium

  * Initial release.

Last-Updated: 2024-02-23
Reviwed-By: Kathara Sasikumar <katharasasikumar007@gmail.com>

Index: moin/setup.py
===================================================================
--- moin.orig/setup.py
+++ moin/setup.py
@@ -92,13 +92,12 @@ setup_args = dict(
         'setuptools>=51',  # dependency with setuptools_scm
         'sqlalchemy<2.0',  # used by sqla store
         'XStatic>=0.0.2',  # support for static file pypi packages
-        'XStatic-Bootstrap==3.1.1.2',
+        'XStatic-Bootstrap>=4.0.0',
         'XStatic-Font-Awesome>=6.2.1.0',
         'XStatic-CKEditor>=3.6.1.2',
         'XStatic-autosize',
         'XStatic-jQuery>=1.8.2',
-        'XStatic-jQuery-File-Upload>=10.31.0',
-        'XStatic-svg-edit-moin>=2012.11.15.1',
+        'XStatic-jQuery-File-Upload',
         'XStatic-JQuery.TableSorter>=2.14.5.1',
         'XStatic-Pygments>=1.6.0.1',
     ],
